kvcached/integration/ollama/interfaces.py

```diff
@@ -0,0 +1,423 @@
+import math
+from typing import List, Optional, Tuple
+
+import torch
+
+from kvcached.kv_cache_manager import KVCacheManager
+from kvcached.tp_ipc_util import start_worker_listerner_thread
+from kvcached.utils import CONTIGUOUS_LAYOUT, PAGE_SIZE, get_kvcached_logger
+from kvcached.vmm_ops import create_kv_tensors
+from kvcached.vmm_ops import init_kvcached as _init_kvcached_impl
+from kvcached.vmm_ops import kv_tensors_created
+from kvcached.vmm_ops import shutdown_kvcached as _shutdown_kvcached_impl
+
+logger = get_kvcached_logger()
+
+_kvcached_initialized: bool = False
+_kvcached_device: Optional[str] = None
+_async_sched: bool = False
+_tp_size: int = 1
+_contiguous_layout: bool = CONTIGUOUS_LAYOUT
+_global_manager: Optional[KVCacheManager] = None
+_kv_cache_allocated: bool = False
+
+
+def _allocate_kv_cache_for_ollama() -> None:
```

Comment:
`_allocate_kv_cache` instead of `_allocate_kv_cache_for_ollama` since this is already within the folder of ollama